wvd-templates/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -99,7 +99,6 @@
         },
         "location": {
             "type": "string",
-            "defaultValue": "",
             "metadata": {
                 "description": "Location for all resources to be created in."
             }
```

Comment:
Should we default these to `"[resourceGroup().location]"` ?

Reply:
The internal code review that prompted this change said "The location parameter of nested templates must not have a defaultValue property" so I'm guessing we shouldn't add one. I'm not sure why they said that but maybe the reason is because we want to generate an error in the case where the main template doesn't pass in a value to the nested template. since if the nested template doesn't get the value from the parent then when a user specifies a value for the location in the parent that is different than the resource group's location then nested template would still use the different location in the resource group.
 
In our case the location value specified in the main template is passed into the nested template, and the main template already has a location default value of [resourceGroup().location]. So I think just removing the default value might be the best option.

Reply:
I see.